client/stylesheets/_toCat.scss

```diff
@@ -4642,10 +4706,93 @@ On click the images will be displayed at normal size to complete the effect
     text-overflow: ellipsis;
 }
 
+/** Social Button CSS **/
+
+.share-btn {
+    display: inline-block !important;
+    color: #ffffff !important;
+    border: none !important;
+    padding: 15px 10px !important;
+    width: 4em !important;
+    box-shadow: 0 2px 0 0 rgba(0,0,0,0.2) !important;
+    outline: none !important;
+    text-align: center !important;
+}
+
+.share-btn:hover {
+  color: #eeeeee !important;
+}
+
+.share-btn:active {
+  position: relative !important;
+  top: 2px !important;
+  box-shadow: none !important;
+  color: #e2e2e2 !important;
+  outline: none !important;
+}
+
+.share-btn.twitter     { background: #55acee !important; }
+.share-btn.google-plus { background: #dd4b39 !important; }
+.share-btn.facebook    { background: #3B5998 !important; }
+.share-btn.linkedin    { background: #4875B4 !important; }
+.share-btn.email       { background: #444444 !important; }
+.share-btn.reddit      { background: #ff5700 !important; }
+
+.share-btn i {
+    color: #fff !important;
+}
+
+.listingItemShare {
+    margin: 10px 0 22px 0;
+}
+
+/* Clearfix */
+
+.clearfix:after {
+     visibility: hidden;
+     display: block;
+     font-size: 0;
+     content: " ";
+     clear: both;
+     height: 0;
+ }
+
+.clearfix { display: inline-block; }
+/* start commented backslash hack \*/
 
+* html .clearfix { height: 1%; }
+.clearfix { display: block; }
+/* close commented backslash hack */
 
 
 
 
+/* Options on Item Page */
+
+.listingItemOptionsDrop {
+    width: 100%;
+    margin-top: 23px;
+}
+
+.listingItemOptionsDrop ul li{
+    width: 100%;
+    padding: 20px 0;
+    background-color: orange;
+    text-align: center;
+    border-radius: 4px;
+}
+
+.listingItemOptionsDrop ul li a {
+    padding: 20px;
+    background-color: orange;
+    color: #fff !important;
+}
 
+.listingItemOptionsDrop ul li a i {
```

Comment:
Merge rule `.listingItemOptionsDrop ul li a i` with rule on line 4772<br>Selector should have depth of applicability no greater than 2, but was 5<br>Selector `listingItemOptionsDrop` should be written in lowercase with hyphens